server_test.go

```
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"bytes"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"log"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+// Creates a new file upload http request with params as cmd arguments.
+func newRequest(uri string, fileKeyName string, params map[string]string, path string) (*http.Request, error) {
+	file, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
```

error is being ignored.

---

Handled